### PR TITLE
make pai daemon with high priority

### PR DESCRIPTION
**Pull request by Binyang2014**
Assign PAI daemon with high priority. Make sure PAI service will not be evicted before user pod.
According to: https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/#evicting-end-user-pods
The kubelet ranks Pods for eviction first by resource, then by priority.
We already changed pai daemon service QoS class to Burstable and make sure it will not exceed the limit resource.

### CODE REVIEW

File: src/cluster-configuration/deploy/priority-class.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: pai-daemon-service-priority
+value: 1000000
+globalDefault: false
+description: "This priority class should be used for PAI daemon service only."
```
Comment (abuccts) [inline src/cluster-configuration/deploy/priority-class.yaml:7]: could you use the built in priority `system-node-critical`?

Comment (Binyang2014) [reply src/cluster-configuration/deploy/priority-class.yaml:7]: This priority is limited to `kube-system` namespace. We can use `system-node-critical`priority in other namespace after 1.1.7 version.

Refer to
https://github.com/kubernetes/kubernetes/issues/60596#issuecomment-559537077

File: src/cluster-configuration/deploy/priority-class.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: pai-daemon-service-priority
```
Comment (abuccts) [inline src/cluster-configuration/deploy/priority-class.yaml:4]: daemon and service are duplicate, pls keep only one term

Comment (Binyang2014) [reply src/cluster-configuration/deploy/priority-class.yaml]: fixed

**Comment by Binyang2014** [comment]
related with: https://github.com/microsoft/pai/issues/4694

**Pull request merged**